youtube-dl-gui.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=youtube-dl GUI
+Exec=youtube-dl-gui
+Icon=youtube-dl-gui
+Categories=Network;Multimedia
```

Comment:
You should also remove the "Mutlimedia" category since `lintian` reports a warning and there is no "Multimedia" category in the FreeDesktop [specification](https://specifications.freedesktop.org/menu-spec/1.0/apa.html).